src/PrestaShopBundle/Model/Product/AdminModelAdapter.php

```diff
@@ -119,6 +119,8 @@ class AdminModelAdapter extends \PrestaShopBundle\Model\AdminModelAdapter
         'low_stock_alert',
         'available_date',
         'ecotax',
+        'additional_shipping_cost',
```

Comment:
It's that simple? 😅
Why wasn't it needed before the fix in ObjectModel?

Reply:
Is it because partial update didn't really work before so everything was saved and now that it works as expected we realize that some fields had been forgotten in this weird multishop feature? 😛

Reply:
Yep, you can follow my investigations in comments of initial issue.
If I well understood different refactoring stages, 
1. Initially, `multishop_check` fields were given in old templating system by a `products.js` script. This script already didn't manage our two fields.
2. When multishop was designed, this code : https://github.com/PrestaShop/PrestaShop/blob/8.0.x/controllers/admin/AdminProductsController.php#L1883-L1885 was used in controller to get fields to update under multishop context but the function you fixed here : https://github.com/PrestaShop/PrestaShop/commit/66a6843bfca2d056896858c5b11a91c85eb16b7f was bugged and in fact our fields went threw this check and were always updated even if not passed by front-end request.
3. Then the templating system has been updated and the adapter class was created. The developer in charge probably used elements he found in script.
4. You fixed the issue of the bugged function and then revealed the initial bug ^^'

1 week of investigations xD